specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -1343,6 +1768,262 @@
+          "uniqueItems": true
+        },
+        "localOutboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Local Outbound profile names from which Outbound is allowed. In current version, it's value is set to ['*'] to allow outbound from all profiles. In later version, user will be able to modify it.",
+          "uniqueItems": true
+        },
+        "remoteInboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Remote Inbound profile names to which Inbound is allowed. Use ['*'] to allow inbound to all profiles. This property can only be updated in auto-approval mode. It's default value is ['*'].",
+          "uniqueItems": true
+        },
+        "remoteOutboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Remote Outbound profile names from which Outbound is allowed. In current version, it's value is set to ['*'] to allow outbound from all profiles. In later version, user will be able to modify it.",
+          "uniqueItems": true
+        },
+        "description": {
+          "type": "string",
+          "description": "A message passed to the owner of the remote NSP link resource with this connection request. Restricted to 140 chars."
+        },
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The NSP link state.",
+          "enum": [
+            "Approved",
+            "Pending",
+            "Rejected",
+            "Disconnected"
+          ],
+          "x-ms-enum": {
+            "name": "nspLinkStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Properties of NSP Link resource."
+    },
+    "NspLinkReconcile": {
+      "type": "object",
+      "properties": {},
+      "description": "Request object for NSP link reconcile."
+    },
+    "NspLinkReference": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NspLinkReferenceProperties",
+          "description": "Properties of the network security perimeter linkReference"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the linkReference resource that is unique within a perimeter. This name can be used to access the resource."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Identifier of the network security perimeter linkReference in ARM id format."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
```

Comment:
These properties should already be there in the definition for ProxyResource

Reply:
Thanks, removed it.